client/components/domains/use-your-domain-step/index.tsx

```diff
@@ -1,13 +1,11 @@
 import { isPlan } from '@automattic/calypso-products';
 import page from '@automattic/calypso-router';
 import { Card, Button, Gridicon } from '@automattic/components';
+import { HELP_CENTER_STORE } from '@automattic/help-center/src/stores';
```

Comment:
Would be great if we had a centrally registered help center store! Right now some of the code initializes their own copy of the store. It might be something to reconsider  as we proceed with #74399 eventually